README.md

```diff
@@ -1,3 +1,33 @@
 # Apache FPM container based on phusion
 Simple apache-vhost that serves content from /var/www/web - php-requests are
 proxied to a linked fpm-container named "fpm" on port 9000.
+
+# mkcert
+
+This image has [mkcert](https://github.com/FiloSottile/mkcert)
+builtin.
+
+Run `mkcert -install` on your host machine.
+
+Then you add the generated CAROOT as a volume (the path on the host
+machine is the output of `mkcert -CAROOT`).
+
+In your `docker-compose.yml` supply one or more host names to be be
+used for HTTPS in either:
+
+1. environment variable `MKCERT_DOMAINS`,
+1. the environment variable `VIRTUAL_HOST`, or
+1. as `hostname` and `domainname` configuration
+
+```yaml
+    volumes:
+      - '${HOME}/Library/Application Support/mkcert:/mkcert/mac:ro'
+      - '${HOME}/.local/share/mkcert:/mkcert/linux:ro'
+
+    environment:
+      MKCERT_DOMAINS: "example.docker *.example.docker local.docker"
+      VIRTUAL_HOST: example.docker
+
+    hostname: example
```

Comment:
The documentation I could find on hostname and domainname was sketchy at best. What is the usecase for this @arnested ?

Reply:
https://docs.docker.com/compose/compose-file/#domainname-hostname-ipc-mac_address-privileged-read_only-shm_size-stdin_open-tty-user-working_dir

It sets the host name and domain name for the container.

The `mkcert.sh` in the container will if neither `MKCERT_DOMAINS` or `VIRTUAL_HOST` environment variables are set, use the output of `hostname -f` (which can be set via those options).